packages/chat-client/src/client.ts

```diff
@@ -387,6 +402,28 @@ export class ChatClient extends IChatClient {
       }
     );
 
+    const historyFetchedStores = [
+      CHAT_THREADS_CONTEXT,
+      CHAT_SENT_INVITES_CONTEXT,
+    ];
+
+    const stores = this.syncClient.storeMap
+      .getAll({ account })
+      .filter((store) => {
+        return historyFetchedStores.includes(store.key);
+      });
+
+    stores.forEach((store) => {
+      const history = new HistoryClient(this.core);
```

Comment:
Seems like we're creating history clients in iterative ways in multiple places here. Is there a risk of this causing memory leaks and/or zombie historyClients that never tear down properly?

Reply:
`HistoryClient` only has 3 members:

1. A pointer to core
2. A string of history URL
3. A string of a `jwt` 

It's essentially a very thin wrapper over 2 utils, so I'm not worried about it personally. 

If you think it's an issue I can refactor it to be long-living, but I don't see the point of keeping it in memory